service-base/src/main/java/org/eclipse/hono/service/credentials/CredentialsHttpEndpoint.java

```diff
@@ -106,6 +98,9 @@ public String getName() {
         return CredentialsConstants.CREDENTIALS_ENDPOINT;
     }
 
+    /**
+     * @deprecated This is removed in the new API.
```

Comment:
I do not think that anybody knows what *the new API* is ...

Reply:
what do you think about a different wording : 

@deprecated This is removed in the official HTTP API.

Reply:
Do we currently have an *unofficial* API? This might be removed in Hono 1.0, if that is the version that you would like the new version of the HTTP API to be part of ...

Reply:
Yes it would be good to have this in Hono 1.0 as it will be a stable version, so I expect we won't be able to do a lot of change afterwards.